Documentation/ExtensionArchitecture/BestPractises/ExtensionLoadingOrder.rst

```diff
@@ -0,0 +1,66 @@
+.. include:: /Includes.rst.txt
+.. index:: Extension development; Software Design Principles
+.. _extension-loading-order:
+
+=======================
+Extension loading order
+=======================
+
+In TYPO3, the order in which extensions are loaded can impact system behavior.
+This is especially important when an extension overrides, extends, or modifies
+the functionality of another. TYPO3 initializes extensions in a defined order,
+and if dependencies are not loaded beforehand, it can lead to unintended
+behavior.
+
+.. _extension-loading-order-composer:
+
+Composer-based installations
+----------------------------
+
+In Composer-based installations, extensions and dependencies are
+installed based on the order specified in the
```

Comment:
Is having `based on the order specified in the composer.json file` hard to read for beginners? Should we take care of adding more specific info here about the general composer.json basic functions with a "read more" link to the official composer.json documentation on packagist?

Reply:
"The order specificied in the composer.json" is ambiguous and IMHO also not a good recommendation. composer.json can have the [`sort-packages`](https://getcomposer.org/doc/06-config.md#sort-packages) option set which would forfeit relying on the order.

I would rephrase that to say 
```suggestion
installed based on the dependencies specified in the
```